common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -314,6 +322,67 @@ public static String simpleClassName(Class<?> clazz) {
         }
     }
 
+    /**
+     * Escapes the specified value, if necessary according to
+     * <a href="https://tools.ietf.org/html/rfc4180#section-2">RFC-4180</a>.
+     * <p>
+     * The {@link CharSequence} will be escaped is if contains a
+     * <ul>
+     *     <li>{@link StringUtil#COMMA}</li>
+     *     <li>{@link StringUtil#LINE_FEED}</li>
+     *     <li>{@link StringUtil#CARRIAGE_RETURN}</li>
+     *     <li>{@link StringUtil#DOUBLE_QUOTE}</li>
+     * </ul>
+     * If a double quote is found inside of the value, it will be escaped by preceeding it
+     * with another double quote.
+     *
+     * @param value the value to be appended, and quoted if necessary
```

Comment:
This reads a bit awkward.  Could we clarify?  How about:

`@param value The value which will be escaped according to <a href="https://tools.ietf.org/html/rfc4180#section-2">RFC-4180</a>`